packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -105,170 +138,170 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget> {
     _update();
   }
 
+  Widget _visualizeWidthAndHeight({
+    @required Widget widget,
+    @required LayoutProperties properties,
+    double arrowHeadSize = defaultArrowHeadSize,
+  }) {
+    return BorderLayout(
+      center: widget,
+      right: Container(
+        child: ArrowWrapper.bidirectional(
+          arrowColor: heightIndicatorColor,
+          arrowStrokeWidth: arrowStrokeWidth,
+          arrowHeadSize: arrowHeadSize,
+          child: RotatedBox(
+            quarterTurns: 1,
+            child: Text(
+              '${properties.describeHeight()}\n'
+              '(${properties.describeHeightConstraints()})',
+              textAlign: TextAlign.center,
+              style: const TextStyle(
+                height: 1.0,
+              ),
+            ),
+          ),
+          direction: Axis.vertical,
+          distanceToArrow: distanceToArrow,
+        ),
+        margin: const EdgeInsets.only(
+          top: margin,
+          left: margin,
+          bottom: widthArrowIndicatorSize,
+        ),
+      ),
+      rightWidth: heightArrowIndicatorSize,
+      bottom: Container(
+        child: ArrowWrapper.bidirectional(
+          arrowColor: widthIndicatorColor,
+          arrowHeadSize: arrowHeadSize,
+          arrowStrokeWidth: arrowStrokeWidth,
+          child: Text(
+            '${properties.describeWidth()}\n'
+            '(${properties.describeWidthConstraints()})',
+            textAlign: TextAlign.center,
+            style: const TextStyle(
+              height: 1.0,
```

Comment:
is the `height: 1.0` needed?

Reply:
yes this is needed so that these two lines are closer and does not take as much space.

With height 1.0: 
![image](https://user-images.githubusercontent.com/25892469/68984815-1c208100-07c7-11ea-9579-b750c0a2dc59.png)


Without height 1.0:
![image](https://user-images.githubusercontent.com/25892469/68984789-01e6a300-07c7-11ea-8d27-c0aff8e432c4.png)